pipelines/managed/rhtap-service-push/README.md

```diff
@@ -24,6 +24,7 @@
 
 ## Changes in 4.8.0
 * Add required taskGit* parameters for collect-slack-notification-params task
+* Add new required params to the rhtap-service-push pipeline for update-infra-deployments task
```

Comment:
Definitely a nit pick but "to the rhtap-service-push pipeline" is pretty obvious.. but we are removing changelogs like next sprint anyways so doesn't matter